### DISCUSSION TITLE
New Source: SAP Sybase IQ

### DISCUSSION THREAD

**Discussion by guilherme-gonzalez** [airbytehq/airbyte#34452] [New Connector Request]
## Connector Name
SAP Sybase IQ 

## What type of integration
Source

## Context

Hello,

We’re currently working with Ericsson (telecom), a vendor that uses SAP IQ for their data storage. Right now, we’re directly connecting to their database to access data (as a data source, using a traditional ETL tool).

We’re planning to switch to Airbyte soon, but we’re a bit concerned about how we’ll get data from Ericsson’s SAP IQ.

Could a connector be developed for this? It would really help streamline our transition.

Thanks!

**Comment by miquelmartorell** [comment]
And SAP SQL Anywhere please ;)

[a comment by an automated account is not shown]
